app/presenters/speech_presenter.rb

```diff
@@ -0,0 +1,39 @@
+class SpeechPresenter < ContentItemPresenter
+  include DateHelper
+
+  def delivery_type
```

Comment:
You could potentially refactor this with a guard clause? (not a blocking comment, just a suggestion):

```
def delivery_type
  return I18n.t("formats.speech.written_on") if content_item.document_type == "authored_article"

  I18n.t("formats.speech.delivered_on")
end
```

Reply:
(Also this is only ever used in the important metadata method, you might make it private)